src/components/interstitials/interstitials.tsx

```diff
@@ -29,6 +30,16 @@ export const Interstitials: FC<Props> = ({
   const backgroundColor = `${color}.main`;
   const characterSplashScreen = getCharacterSplashScreen(name);
   const characterTitle = getCharacterTitle(name);
+  const playSfx = usePlaySFX();
+
+  useEffect(() => {
+    if (splashScreenVariant === "chat") {
+      void playSfx("./music/voice-over/StartChating.mp3");
+    }
+    if (splashScreenVariant == "voting") {
```

Comment:
`==` => `===`

Reply:
You could also conditionally define the value for the `playSfx` arg and only declare the call once. It's a bit more clear.